.travis.yml

```diff
@@ -6,7 +6,7 @@ dist: trusty
 addons:
   postgresql: "9.5"
 env:
-  - TRAVIS_NODE_VERSION="7.7.3"
+  - TRAVIS_NODE_VERSION="6.9.0"
```

Comment:
Was travis ci breaking because we were running to recent a version?

Reply:
Yes it was.